src/reader/timing/word-analysis.ts

```
@@ -0,0 +1,133 @@
+  she: 293048,
+  do: 289925,
+  how: 289414,
+  their: 285391,
+  if: 284992,
+  will: 256933,
+  up: 254545,
+  other: 236431,
+  about: 235524,
+  out: 233949,
+  many: 230372,
+  then: 229761,
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)